Protocol rename to Microsoft.Testing.Platform protocol (#6596)
microsoft/testfx: docs/mstest-runner-protocol/001-protocol-intro.md

> # 001 - Microsoft.Testing.Platform Protocol (formerly MSTest Runner Protocol)

Reviewer: ```suggestion
# 001 - Microsoft.Testing.Platform JSON-RPC Protocol (formerly MSTest Runner Protocol)
```

> Microsoft.Testing.Platform projects builds into a self-contained executable that can be invoked to run tests.

Author: ```suggestion
Microsoft.Testing.Platform projects builds into an executable that can be invoked to run tests.
```

> Microsoft.Testing.Platform projects builds into a self-contained executable that can be invoked to run tests.

Author: Which one do you prefer @Youssef1313 ?

> Microsoft.Testing.Platform projects builds into a self-contained executable that can be invoked to run tests.

Reviewer: No objection to the first one, but I think the second suggestion is simpler and is clear.
